morphman/manipulate_area.py

```diff
@@ -376,6 +372,58 @@ def change_area(voronoi, factor, line_to_change, diverging_centerline, diverging
     return new_voronoi, new_centerlines
 
 
+def get_asymmetric_displacement(A, angle_asymmetric, factor_, frenet_normals_array, frenet_tangents_array, locator,
+                                point):
+    """
+    Compute translation adding asymmetric effect to stenosis / bulge
+
+    Args:
+        A (ndarray): Initial position on centerline
+        angle_asymmetric (Angle to rotate around Frenet tangent vector:
+        factor_ (float): Scaling factor for area manipulation
+        frenet_normals_array (list): List of Frenet normal vectors
+        frenet_tangents_array (list): List of Frenet tangent vectors
+        locator (vtkPointLocator): Centerline locator
+        point (vtkPoint): Current Voronoi point
+
+    Returns:
+        ndarray: Translation vector for asymmetric effect
+    """
+    cl_id = locator.FindClosestPoint(point)
+    frenet_normal = frenet_normals_array[cl_id]
+    frenet_normal_vector = A + frenet_normal
+    frenet_normal_vector /= la.norm(frenet_normal_vector)
+    frenet_tangent = frenet_tangents_array[cl_id]
+    R = get_rotation_matrix(frenet_tangent, angle_asymmetric)
+    v = np.dot(R, frenet_normal_vector) * (1 - factor_) * 1.5
```

Comment:
What is the hard-coded 1.5 value here?

Reply:
This is also a mystery to me. However, without this value there remains a tiny gap between the stenosed part and the initial model wall. 

I'd like to investigate this further, before merging.